init.yaml

```diff
@@ -15,6 +19,8 @@ secrets:
         value: admin
       - name: basic-auth-password
         value: ""
+    filters:
+      - "auth"
```

Comment:
I would use some help with what filters secrets should have and the proper naming